granite/toBig.py

```diff
@@ -16,13 +16,10 @@
 #
 #################################################################
 import sys, os
-import ctypes
```

Comment:
I did not review this file in detail because I think I'm being asked to review it for the changes in poetry, not the other parts. And I'm on vacation, so I have limited time. Please make sure another reviewer reviews this file.